app/assets/v2/js/grants/fund.js

```diff
@@ -202,3 +187,12 @@ const updateSummary = (element) => {
     $('#summary-frequency').html($('input#period').val());
   });
 };
+
+const enableWaitState = () => {
```

Comment:
coulkd maybe go into a shared file so it can be re-used?

Reply:
coolio